### PR TITLE
OSASINFRA-3483: Add openstack-*-csi jobs for cluster-storage-operator

### PR DESCRIPTION
**Pull request by stephenfin**
So that we can test our Hypershift-inspired changes going forward.

Signed-off-by: Stephen Finucane <stephenfin@redhat.com>

### CODE REVIEW

**Comment by EmilienM** [comment]
/lgtm
/pj-rehearse ack
I don't think we can rehearse the jobs yet.

File: ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml [hunk 1]
```diff
@@ -153,6 +153,19 @@ tests:
       CONFIG_TYPE: minimal
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-cinder-csi
```
[the remark this answers is not in the document]
Comment (stephenfin) [reply ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml]: Done.

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-cluster-storage-operator-master-e2e-openstack-cinder-csi
/pj-rehearse pull-ci-openshift-cluster-storage-operator-master-e2e-openstack-manila-csi

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-cluster-storage-operator-master-e2e-openstack-manila-csi

**Pull request merged**